persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -619,7 +619,7 @@ mayDefault def = case def of
 
 type SafeToRemove = Bool
 
-data AlterColumn = Type SqlType Text
+data AlterColumn = ChangeType SqlType Text
```

Comment:
In the course of debugging this issue I found it extremely hard to search for `Type`, so I renamed this to `ChangeType`.